src/app/layout.tsx

```diff
@@ -0,0 +1,264 @@
+import '../styles/global.css'
+
+import { Toaster } from '@/components/ui/toaster'
+import { client } from '@/gql/client'
+import { siteDataQuery } from '@/gql/query'
+
+import { GeistSans } from 'geist/font/sans'
+import { Metadata } from 'next'
+import localFont from 'next/font/local'
+
+// Font files can be colocated inside of `pages`
+const menloFont = localFont({ src: '../lib/fonts/Menlo-Regular.woff', variable: '--font-menlo' })
+
+const DESCRIPTION =
```

Comment:
Did we have to move the layout homepage to the app router as well as part of this optimization for loading delegate page ?

Reply:
We need to move at least the `_app.ts` (page router) to `layout.tsx` (app router) but the page itself I can move it back to the page router